jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/JKubeEnricherContext.java

```diff
@@ -94,10 +94,7 @@ public JKubeEnricherContext(
             .processorConfig(processorConfig)
             .jKubeBuildStrategy(jKubeBuildStrategy)
             .pluginConfigLookup(
-                (system, id) -> {
```

Comment:
Is it possible we might still want to distinguish the plugin system we're using ---> `maven`, `gradle`, and whatever comes next?
What was the motive to remove this differentiation? is it redundant?